### DISCUSSION TITLE
Telekom MagentaTv

### DISCUSSION THREAD

**Discussion by frgtv10** [fabianishere/udm-iptv#8] [Q&A]
Hi guys,

I have a UDM Pro (Beta Firmware 1.11.0) with USW Switch 24 POE.
UDM Pro WAN is connected to the Telekom Glass FIber FTTH.
 
I installed everything as described. Telekom IPTV is on VLAN 7, as the other internet traffic as well.
It doesnt seem really to work, maybe some configuration is wrong?
TV image will froze after 10 seconds...

My config is:
`IPTV_WAN_INTERFACE="ppp0"
IPTV_WAN_RANGES="232.35.0.0/16 239.35.0.0/16 224.0.0.0/4"
IPTV_WAN_VLAN="7"
IPTV_WAN_DHCP_OPTIONS="-O staticroutes -V IPTV_RG"
IPTV_LAN_INTERFACES="br0"`

After starting:

`Starting container...
a98ac9cf71cb5be8f2bfb77628ebd572a38cdd8c85b122fd0269571a4edf81cf
WARNING: The same type, major and minor should not be used for multiple devices.
WARNING: The same type, major and minor should not be used for multiple devices.
WARNING: The same type, major and minor should not be used for multiple devices.
WARNING: The same type, major and minor should not be used for multiple devices.
WARNING: The same type, major and minor should not be used for multiple devices.
WARNING: The same type, major and minor should not be used for multiple devices.
WARNING: The same type, major and minor should not be used for multiple devices.
5a8c4d86947ec0e33a1476f68702f505c26ac41865347343d35ff4f526a34dcc
udm-iptv: NATing IPTV network ranges (if necessary)...
udm-iptv: Setting up igmpproxy..
udm-iptv: Starting igmpproxy..`

And also the troubleshooting info doesnt show any information:
`ip -4 addr show dev iptv`

DHCP Options are activated, IGMP Snooping is activated.
Thank you for you help and time.

Regards
Matt

**Comment by thomas-baumeister** [comment]
Since I don't have Telekom Glass FIber FTTH, I don't know your setup in detail. 
Essentially, it does not make sense to set IPTV_WAN_VLAN to 7 and have the DHCP options. Since you already have configured your internet connection to use VLAN 7 you probably get this device error because of this.

Anyway, Fabians instructions are mostly geared to KPN customers. For Telekom Entertain you'll find https://github.com/fabianishere/udm-kernel-tools/issues/2 a good read. It boils down to the following start up script:

```
if podman container exists iptv; then
  podman rm -f iptv
fi

podman run --network=host --privileged \
    --name iptv -i -d --restart always \
    -e IPTV_WAN_INTERFACE="ppp0" \
    -e IPTV_WAN_VLAN="0" \
    -e IPTV_WAN_RANGES="232.0.0.0/16 87.141.0.0/16" \
    -e IPTV_LAN_INTERFACES="br10" \
    fabianishere/udm-iptv:1.1 -d -v
```
You just may need to set your LAN range with `IPTV_LAN_RANGES=""
`

Best regards
Thomas

**Comment by TheJulianJES** [comment]
Try reading this (closed) issue: https://github.com/fabianishere/udm-kernel-tools/issues/2
Note: The earlier comments are all reading the "bare metal" installation of igmpproxy in the unifi-os container.
The newer comments should refer to the newer IPTV docker container. Hopefully something will help you! (Maybe the firewall rules, ...)

**Reply by frgtv10** [reply]
Thank you so much... the ****ing firewall was the problem.
After adding the two ips (232.0.0.0/16 87.141.0.0/16) and allow it it works.

But after 2-5 Minutes I got a stocking image, but it takes back after around 3-4 seconds. Maybe something is still wrong here?

Maybe somebody got a list of necessary IPs?

**Reply by TheJulianJES** [reply]
Did you correctly add all four rules from here: https://www.bjoerns-techblog.de/2017/06/deutsche-telekom-entertain-mit-dem-ubiquiti-unifi-security-gateway/
I think I remember having the issue with stutter after 3—4 minutes because of not having all firewall rules.

**Comment by stan23** [comment]
Hi,
thanks a lot for these hints! I got MagentaTV working with UDM/P.

For reference I summarized my settings:

I have a UDM Pro running UniFi OS Version 1.11.0-20 (early access) because that kernel already supports multicast routing.
My provider is Deutsche Telekom with 50 Mbit VDSL, so IPTV and internet are received on VLAN 7.

Modem: Vigor Draytek set up according to [1]
Firewall rules are set up according to [1]

WAN network settings are PPPoE with VLAN ID 7.


My settings for the container:
```
# cat /mnt/data/on_boot.d/15-iptv.sh
IPTV_WAN_INTERFACE="ppp0"
IPTV_WAN_RANGES="232.0.0.0/16 87.141.0.0/16"
IPTV_WAN_VLAN="0"
IPTV_WAN_DHCP_OPTIONS="-O staticroutes -V IPTV_RG"
IPTV_LAN_INTERFACES="br0"

if podman container exists iptv; then
  podman rm -f iptv
fi
podman run --network=host --privileged \
    --name iptv -i -d --restart on-failure:5 \
    -e IPTV_WAN_INTERFACE="$IPTV_WAN_INTERFACE" \
    -e IPTV_WAN_RANGES="$IPTV_WAN_RANGES" \
    -e IPTV_WAN_VLAN="$IPTV_WAN_VLAN" \
    -e IPTV_WAN_DHCP_OPTIONS="$IPTV_WAN_DHCP_OPTIONS" \
    -e IPTV_LAN_INTERFACES="$IPTV_LAN_INTERFACES" \
    -e IPTV_LAN_RANGES="" \
    fabianishere/udm-iptv
```


[1] https://www.bjoerns-techblog.de/2017/06/deutsche-telekom-entertain-mit-dem-ubiquiti-unifi-security-gateway/

**Reply by leifbln** [reply]
Hi guys, 

thanks for your work to make IPTV happen on the UDM.

Unfortunately i followed your guidance and was not able to use IPTV with MagentaTV on my UDM.

The Stream is stopping after 10-15 seconds.

My Setup is the following:

Speedport 4 from Deutsche Telekom in Modem-Mode -> TP-Link Managed Switch with Tagged VLAN7 -> Unifi Dream Machine -> Magenta TV Receiver MR401

The only difference I experienced was that the command ip -4 addr show dev iptv was not bringen me an Output:

ip -4 addr show dev iptv
Device "iptv" does not exist.
Think this should be there?

If I use podman logs iptv I can see lot of entries with the Magenta TV Addresses.

Would be great if you could advise if i'm doing something wrong.

Firewallrules are in place like described here:

(https://www.bjoerns-techblog.de/2017/06/deutsche-telekom-entertain-mit-dem-ubiquiti-unifi-security-gateway)

But also tested with Any to Any Rule. No Success..

MagentaTV Receiver is telling me, that my Network is using old version of IGMP. So I think the Protocol is not working well in my Network.

Thanks,

Leif

**Reply by fabianishere** [reply]
Could you share the configuration you are using (like those posted above)?

**Reply by leifbln** [reply]
`# cat /mnt/data/on_boot.d/15-iptv.sh
IPTV_WAN_INTERFACE="ppp0"
IPTV_WAN_RANGES="232.0.0.0/16 87.141.0.0/16"
IPTV_WAN_VLAN="0"
IPTV_WAN_DHCP_OPTIONS="-O staticroutes -V IPTV_RG"
IPTV_LAN_INTERFACES="br110"

if podman container exists iptv; then
  podman rm -f iptv
fi
podman run --network=host --privileged \
    --name iptv -i -d --restart on-failure:5 \
    -e IPTV_WAN_INTERFACE="$IPTV_WAN_INTERFACE" \
    -e IPTV_WAN_RANGES="$IPTV_WAN_RANGES" \
    -e IPTV_WAN_VLAN="$IPTV_WAN_VLAN" \
    -e IPTV_WAN_DHCP_OPTIONS="$IPTV_WAN_DHCP_OPTIONS" \
    -e IPTV_LAN_INTERFACES="$IPTV_LAN_INTERFACES" \
    -e IPTV_LAN_RANGES="" \
    fabianishere/udm-iptv -d -v
#
`

**Reply by leifbln** [reply]
I'm wondering if the connection between the Modem and the UDM over the managed Switch can cause issues. 

Speedport 4 is just handling the Modem capabilities. The Internal Port of the Modem is plugged into a managed switch from TP-Link on Tagged Port with VLAN 7. From there its going to a second managed Switch who is handling the access port for UDM with a Tagged Port in VLAN 7. Will try later to connect the Modem direct to the UDM without any switch. 

IGMP Snooping on the Switches is enabled.

**Comment by thomas-baumeister** [comment]
Essentially, VLAN tagging should not be an issue, because I assume that you get access to the Internet. 
I would check the firewall rules. I had this in the past as well. There should be 4 rules - two for Internet In, two for Internet local.

**Comment by leifbln** [comment]
![image](https://user-images.githubusercontent.com/95283373/144047540-2fc2c8c3-4559-43d8-9ca9-a29c0a8a7851.png)
![image](https://user-images.githubusercontent.com/95283373/144047575-4a0ff325-b9e8-459c-9b75-10485b054be5.png)
![image](https://user-images.githubusercontent.com/95283373/144047692-778f7053-7ec7-4003-8fb1-550a3743e3d9.png)
![image](https://user-images.githubusercontent.com/95283373/144047725-daf70013-ebc0-4f42-89a7-1574d53b78d2.png)

Rules are looking good. Also extended the Firewall Groups with some additional IPs.

![image](https://user-images.githubusercontent.com/95283373/144047799-e217a16f-1e85-437c-8be7-6221d7f4697a.png)
![image](https://user-images.githubusercontent.com/95283373/144047825-0d16dfb9-4d07-4000-89ad-f9f93d234932.png)
![image](https://user-images.githubusercontent.com/95283373/144047857-61a3f1c0-0645-4550-a4e9-5f3727609496.png)

**Comment by thomas-baumeister** [comment]
Your IGMP Proxy (Internet Local) rule has a set source. It should be any/any.

**Reply by leifbln** [reply]
No change at all :( I already tried so much things but it isnt getting better.

![image](https://user-images.githubusercontent.com/95283373/144049791-1c81ec33-9a86-431a-91c7-9f3f872dd1b5.png)

**Reply by thomas-baumeister** [reply]
Can you clear the destination port group as well? I've overlooked this before.

**Reply by leifbln** [reply]
Same. 

Added two Rules for Internet In and Internet Local with Any <-> Any, basically opened my complete Internet Interface. But experienced the same issue. I don't think that the Firewall is causing this. 

Any other Troubleshooting mechanism to check the firewall logs or even logs for the IGMP Proxy? with podman logs iptv I see the following, while 192.168.110.153 is the Receiver and 110.1 is the UDM:

![image](https://user-images.githubusercontent.com/95283373/144051118-fce9e896-1c2d-4a31-ba6a-66c0ca32a40b.png)

**Reply by fabianishere** [reply]
Are you sure it is necessary to have these firewall rules?

For KPN, it is only necessary to setup a NAT rule for the IPTV addresses, but this is done automatically by the container.

**Reply by leifbln** [reply]
I dont think that this is the Firewall. 

Meanwhile i've tested without the Switches involved. Direct Connection from Modem to UDM and from UDM to Receiver. Same behavior.

Please find attached the Diagnosis from your new version:


/mnt/data/on_boot.d/15-iptv.sh diagnose
Please share the following output width the developers:
=== Configuration ===
WAN Interface: ppp0
WAN Ranges: 232.0.0.0/16 87.141.0.0/16
WAN VLAN: 0
LAN Interfaces: br0
=== IP Link and Route ===
108: ppp0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1492 qdisc pfifo_fast state UNKNOWN group default qlen 3
    inet 79.xxx.xxx.xxx peer 62.155.240.62/32 scope global ppp0
       valid_lft forever preferred_lft forever
62.155.240.62 proto kernel scope link src 79.xxx.xxx.xxx
=== Container Logs ===
udm-iptv: NATing IPTV network ranges (if necessary)...
udm-iptv: Setting up igmpproxy..
udm-iptv: Starting igmpproxy..

**Reply by fabianishere** [reply]
I notice you are using a different VLAN now (`br0` as opposed to `br110`), is that correct?

**Reply by leifbln** [reply]
Had to switch the internal network due to cabling. Not as many long cables as I need :) Behavior is the same with both.

Br0 is the System LAN where in total 5 devices are inside, like Tado, Philips Hue and the Switches. 

Br110 could be the IPTV LAN. This would only have the Receiver inside.

**Reply by leifbln** [reply]
Any advise here, what else I could check or test?

**Reply by fabianishere** [reply]
I recommend capturing a packet trace of your network, to find out what's causing the issue.

**Reply by leifbln** [reply]
Did some Port Mirroring on the WAN Port of the UDM and the LAN Port:

![image](https://user-images.githubusercontent.com/95283373/144289529-0099e4d4-b116-485b-82a8-1e76287534b6.png)

![image](https://user-images.githubusercontent.com/95283373/144290474-bf0091ac-be5b-44c3-abcf-72c1f1e8dc58.png)


Interesting that the LAN Side is running IGMPv2, while the WAN Side is running IGMPv3. 

192.168.2.151 is my PC Interface. 192.168.2.153 is the Media Receiver.

**Reply by leifbln** [reply]
@fabianishere i forgot about one switch in the network which was also involved as I tried my direct connection tests. 

As soon as the Modem is connected via this switch to the UDM it starts with these outages. Removed this switch now and it works. 

But there was an additional thing which could be more or less a Layer-8 Problem but maybe also handled by the script? 

If the UDM gets a new Public IP address WITHOUT Reboot, you need to run the Script again. Is it possible to insert a check if a new Public IP is assigned?

**Reply by gobon1** [reply]
I have the exact same issue. I did all of the steps in the guide. if i check for ip -4 addr show dev iptv it tells me there is no such a device.

Hope we will find a solution.

**Reply by fabianishere** [reply]
The configuration for MagentaTV does not create an iptv device, since IPTV device is carried over the same VLAN as internet.

**Reply by leifbln** [reply]
I agree with fabian. 

What issues are you experiencing? Stop after a few seconds? 

Did you tried to connect the Receiver direkt with the UDM, without any additional Device connected?

**Reply by gobon1** [reply]
Hey Guys,

yes it works and after like 10 seconds it stops. I also tryed to connect the Magenta TV directly to the UDM. Same problem.

I have the Firewall Rules, i clicked through the script and did everything right (I Guess)

`## IPTV Configuration
IPTV_WAN_INTERFACE="ppp0"
IPTV_WAN_RANGES="232.0.0.0/16 87.141.0.0/16 193.158.35.251"
IPTV_WAN_VLAN="0"
IPTV_WAN_DHCP_OPTIONS="-O staticroutes -V IPTV_RG"
IPTV_LAN_INTERFACES="br0"
IPTV_IGMPPROXY_ARGS=""
`

DHCP Options set. But i have one Network for all devices. Should i try an separate Network for the Magenta Receiver?

BR
Gobon

**Reply by fabianishere** [reply]
Are you using the correct firmware on the UDM? That is, version 1.11.0+?

**Reply by gobon1** [reply]
oh no, i got 1.10.4 and Network 6.5.55.

I startet now Beta releases. I guess it is a beta?

**Reply by fabianishere** [reply]
Yes

**Reply by gobon1** [reply]
I did assign to beta Programm but i only can use the official Release Channel. How long does it take if i can use the Beta program on my to mi Ui account assigned UDM?

On my UI account it tells me that my Early-access-Programm is active

**Reply by leifbln** [reply]
Just update like described here:

https://community.ui.com/releases/UniFi-OS-Dream-Machines-1-11-0-23/0f185960-05dc-4332-a30c-dc63a3c3ba2d

Manual upgrade instructions

SSH into your UDM-Base/UDM-Pro and execute the following in it's shell (replace <firmware_url> with one of the download links found below):

ubnt-upgrade <firmware_url>

UDM-Base <firmware_url>: https://fw-download.ubnt.com/data/udm/ca82-udm-1.11.0-23-4d6ffe9df75043d3844163b15ea95cbb.bin

UDM-Pro <firmware_url>: https://fw-download.ubnt.com/data/udm/2e16-udmpro-1.11.0-23-ea410628dea94e08a7fec5d61e75d777.bin

**Reply by gobon1** [reply]
i don't have Access to this page, i found this link serveral times. 
i Updatet the UDM now on 1.11.0-23. IGMP Snooping active on my Lan Network. Magenta TV directly connected to eth0 on my UDM. But it still starts lagging after 5-10 seconds.

BR
Gobon

**Reply by leifbln** [reply]
Please check if the IP address for the NAT is set correctly (re-run the Installation). 

Additional to that check again the Firewall rules. 

Did you unplugged everything from the UDM and just did a direct connection from Modem -> UDM WAN and Receiver -> UDM LAN (eth0 in your case)

**Reply by gobon1** [reply]
Hey, 

i re-run the installation and had one Firewall rule disabled 🙃 
now MagentaTV is running <3 

special thanks to @fabianishere awesome!
and thanks to @leifbln for the additional support!

best wishes 
Gobon

**Reply by FireSale14** [reply]
@gobon1 
Hi,
what Firewall rule your desabled? My Magenta TV is not running

LG
Michael

**Reply by mangel8** [reply]
Hi did you tried to choos No Vlan in the installation? That was my problem.
Greetings

**Comment by mangel8** [comment]
Hi,

 

have you tried to reinstall the script and choose the korrect VLAN where your TV Box is located


IPTV_LAN_INTERFACES="br0"

And updated the FW Rules ot the IPTV Destination

 

Greetings


Michael
 

Gesendet: Donnerstag, 27. Januar 2022 um 11:46 Uhr
Von: "gobon1" ***@***.***>
An: "fabianishere/udm-iptv" ***@***.***>
Cc: "mangel8" ***@***.***>, "Comment" ***@***.***>
Betreff: Re: [fabianishere/udm-iptv] Telekom MagentaTv (Discussion #8)


 

Hey,

as i had Magenta TV Running it now stopped working. 10 Seconds i get fluent signal and than it Stops.
I Added other Networks such as IoT to move my Hue Magenta Tv and Stuff like that into it. After i moved it to the IoT with VLAN 3 the Magenta TV does not work anymode. Even if i put it back to my prior LAN network.
What do i have to do that this works again? Is it not possible to set the Magenta TV Receiver in an serperatet LAN?

Do i need an Container now? Is this the Right Documentation now? #2 (comment)

I probably need to remove the old installation with apt remove dialog igmpproxy udm-iptv ?

I am kinda depressed that it stopped working even when i put it back now. Its not that easy to understand for me whats needed to do now.

Maybe someone can tell me the requiered steps.

BR
Gobon

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by gobon1** [reply]
Hey,

yes i installed the package now with choosing the IoT network as LAN interface.
Does not help here.

I have a Firewall Rule as Destination Group but did not changed anything here
![image](https://user-images.githubusercontent.com/96193557/151356742-463f04d1-a022-4e24-bace-224355b19f70.png)
![image](https://user-images.githubusercontent.com/96193557/151356833-f1568fb2-b1b6-47cd-a804-be9c3f764967.png)

**Reply by fabianishere** [reply]
Please share the output of `udm-iptv-diag`.

**Reply by gobon1** [reply]
=== Configuration ===
WAN Interface: eth4.7
WAN Ranges: 244.0.0.0/4 87.141.0.0/16 193.158.0.0/15
WAN VLAN: 0 (dev iptv)
WAN DHCP options: -O staticroutes -V IPTV_RG
LAN Interfaces: br3
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
=== Service Logs ===
Jan 27 13:17:49 ubnt systemd[1]: udm-iptv.service: Consumed 194ms CPU time.
Jan 27 13:17:49 ubnt systemd[1]: Started IPTV support for the UniFi Dream Machin                                                                                                             e.
Jan 27 13:17:49 ubnt udm-iptv[30311]: NATing IPTV network ranges (if necessary)
Jan 27 13:17:49 ubnt udm-iptv[30311]: Setting up igmpproxy
Jan 27 13:17:49 ubnt udm-iptv[30311]: Starting igmpproxy
Jan 27 13:17:49 ubnt udm-iptv[30311]: adding VIF, Ix 0 Fl 0x0 IP 0x0103a8c0 br3,                                                                                                              Threshold: 1, Ratelimit: 0
Jan 27 13:17:49 ubnt udm-iptv[30311]: There must be at least 1 Vif as upstream.
Jan 27 13:17:49 ubnt systemd[1]: udm-iptv.service: Main process exited, code=exi                                                                                                             ted, status=255/EXCEPTION
Jan 27 13:17:49 ubnt systemd[1]: udm-iptv.service: Failed with result 'exit-code                                                                                                             '.
Jan 27 13:17:49 ubnt systemd[1]: udm-iptv.service: Consumed 187ms CPU time.

**Reply by fabianishere** [reply]
As far as I know, the WAN interface for MagentaTv should be `ppp0`. Could you try changing to that?

**Reply by gobon1** [reply]
i dont have this option
![image](https://user-images.githubusercontent.com/96193557/151358119-251d953c-5e75-470d-a115-318ba6bf39ba.png)

**Reply by fabianishere** [reply]
Hmm, do you still have your old configuration somewhere in `/mnt/data/on_boot.d` (e.g., `/mnt/data/on_boot.d/15-iptv.sh`)?

**Reply by fabianishere** [reply]
Seeing your old comments, you seem to be using `ppp0`. You can force it to use that interface as follows (in UniFi OS):

```bash
echo "udm-iptv udm-iptv/wan-interface select ppp0" | debconf-set-selections
```

**Reply by gobon1** [reply]
Yes in that file 15-iptv.sh it says ppp0 
i just send echo "udm-iptv udm-iptv/wan-interface select ppp0" | debconf-set-selections via putty in that Unify OS?
after using udm-iptv-diag it still says Eth 4.7

i think i have two kinds of installtions running isnt it?

I have this first Installation i mentioned above. Than i added my IoT lan and now its not working anymore. Now i added with this description https://github.com/fabianishere/udm-iptv/issues/2#issuecomment-1007413230 the container.

What do i need to edit? Now or is the Installation now messed up?

BR
Gobon

**Reply by gobon1** [reply]
I would set up my UDM again if this is easyer to support for you guys. I dont know if this will work with both configurations and how to delete them. I know its hard to explain when someone has no experience in this. If you say its not usefull to set up my UDM again i would leve it as it is.

So i would set up my UDM with 5 Networks
LAN
Guest
IoT
Megenta
VPN

after i set up the UDM i add an Container such as in the tutorial and try to use ppp0 connection. 
After this i set up the Groups of Destination / Source / Port and set the Firewall rules as before.
Than i add the DHCP Options for the Magenta Network.

But what means this below? Or where can i find this file? Should this script be locatet in `/mnt/data/on_boot.d` ?
This is out of the documentation from https://github.com/fabianishere/udm-iptv/issues/2#issuecomment-1007413230

"The proxy will only start once the ppp connection is up, this happens a while after the S15 boot execution.
I renamed the script to 95-iptv.sh and added these lines before the "podman run" line to trigger/delay the connection:

> sysctl net.ipv4.conf.all.force_igmp_version=3
> ping -w 5 google.de
> sleep 30

If the proxy comes up too early, you will see messages about "VIF interfaces", just restart the container manually.
The rename/ping/sleep is my first attempt to fix that on boot.

With my (1.11) UDM I also needed to add firewall policies to allow inbound UDP and IGMP from the IPTV_WAN_RANGES on the "WAN in" firewall section. Of course IGMP snooping has to be enabled.
The "sysctl net.ipv4.conf.all.force_igmp_version=3" above is necessary for the MediaReceiver as it requires IGMPv3, the MagentaTV app doesn"t need this."

BR
Gobon

**Reply by fabianishere** [reply]
The container and the script in `/mnt/data/on_boot.d` represent the old version of this project. Just uninstall it by removing the `15-iptv.sh` script.

Then, set `IPTV_WAN_INTERFACE` to `ppp0` in `/etc/udm-iptv.conf` (enter UniFi OS first).

Finally, reboot your device and check whether it works.

**Reply by gobon1** [reply]
Hey,

i have no such file like "udm-iptv.conf" in /etc.

BR
Gobon

**Reply by fabianishere** [reply]
It resides in UniFi OS. You need to enter it first: `unifi-os shell`.

**Reply by gobon1** [reply]
I opened the UDM via WinSCP. How do i open the file from the shell? Nano is not working.

**Reply by gobon1** [reply]
I don't find an text editor in unify OS. Is there any?

Br
gobon

**Reply by fabianishere** [reply]
```bash
apt install nano
```

**Reply by gobon1** [reply]
It Works! 
i feel a little retarded now that i dont thought about installing what i muss :D Since a while i hade to do with Linux :D 

Thank you so much.. AGAIN! Hope i need no further help :D

**Reply by gobon1** [reply]
Hey @fabianishere 

yesterday did my UDM an Auto Update on 1.11.4 and the IPTV did not work anymore. 

I triggered a new installation to set all parameters again and now it works.

Is there a way to save the config or the Container through firmware upgrades?

BR,
Felix

**Comment by itobi1** [comment]
Hello @fabianishere and everybody else,
I configured my UDM Pro as mentioned in this blog and everything works fine so far, except for my Telekom Media Receivers.
I've read, that I have to configure "sysctl net.ipv4.conf.all.force_igmp_version=3" but it doesn't persist through restarts.
Does anybody have a solution for this?

Thanks and best regards
itobi1

**Reply by itobi1** [reply]
I could make it to set the vartriable on startup but it still uses IGMPv2. Any suggestions?

# sysctl net.ipv4.conf.all.force_igmp_version
net.ipv4.conf.all.force_igmp_version = 3
# tcpdump igmp
tcpdump: verbose output suppressed, use -v or -vv for full protocol decode
listening on switch0, link-type EN10MB (Ethernet), capture size 262144 bytes
17:12:39.353993 IP 10.0.2.2 > 239.255.255.250: igmp v2 report 239.255.255.250
17:12:39.597931 IP 10.0.2.2 > 232.0.20.140: igmp v2 report 232.0.20.140
